package.json

```diff
@@ -37,14 +37,12 @@
     "sinon": "^2.1.0",
     "sleep": "^6.1.0",
     "uglifyjs-webpack-plugin": "^0.4.3",
-    "webpack": "^2.3.3"
-  },
-  "optionalDependencies": {
+    "webpack": "^2.3.3",
     "fsevents": "^1.2.9"
```

Comment:
Is this correct? I remember you had problems installing fsevents so that's why we moved it to optional